machine-learning/Dockerfile

```diff
@@ -1,34 +1,27 @@
-FROM node:16-bullseye-slim as builder
-
-ARG DEBIAN_FRONTEND=noninteractive
+# first stage
+FROM python:3.10 AS builder
 
 WORKDIR /usr/src/app
 
-RUN apt-get update
-RUN apt-get install gcc g++ make cmake python3 python3-pip -y
-
-COPY package.json package-lock.json ./
-
-RUN npm ci
-RUN npm rebuild @tensorflow/tfjs-node --build-from-source
-
+# COPY requirements.txt .
 COPY . .
 
-FROM builder as prod
-
-RUN npm run build
-RUN npm prune --omit=dev
-
-FROM node:16-bullseye-slim
-
-ARG DEBIAN_FRONTEND=noninteractive
+# install dependencies to the local user directory (eg. /root/.local)
+# RUN pip install --user -r requirements.txt
+RUN pip install --user --no-cache-dir torch==1.13.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip install --user transformers tqdm numpy scikit-learn scipy nltk sentencepiece flask Pillow
+RUN pip install --user --no-deps sentence-transformers
 
+# prod stage
+FROM python:3.10-slim as prod
 WORKDIR /usr/src/app
```

Comment:
Near the top here, can you add `ENV TRANSFORMERS_CACHE=/cache` to specify a default for the model cache directory? ([docs](https://huggingface.co/docs/transformers/installation?highlight=transformers_cache#cache-setup))